numpy/core/tests/test_multiarray.py

```diff
@@ -6355,50 +6360,52 @@ def test_string(self):
         assert_equal(np.where(False, b, a), "abcd")
 
 
-class TestSizeOf(TestCase):
+if not IS_PYPY:
```

Comment:
Interesting suggestion. Probably not as obvious in reading the code, but otherwise cleaner, at least until we drop the now abandoned Nose.